packages/metal-dom/src/features.js

```diff
@@ -56,22 +55,6 @@ class features {
 		}
 		return `${type}end`;
 	}
-
-	/**
-	 * Some browsers (like IE9) change the order of element attributes, when html
```

Comment:
The comment says IE9, but we should check in the real browsers to see which ones actually do this.